frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostGroupsModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { t } from 'i18next';
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { MultiSelectDialog, usePageDialog } from '../../../../../framework';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { useAwxView } from '../../../common/useAwxView';
+import { InventoryGroup } from '../../../interfaces/InventoryGroup';
+import { useHostsGroupsColumns } from './hooks/useHostsGroupsColumns';
+import { useHostsGroupsFilters } from './hooks/useHostsGroupsFilters';
+
+export interface InventoryHostGroupsAddModalProps {
+  onAdd: (items: InventoryGroup[]) => void;
+}
+
+export function InventoryHostGroupsAddModal(props: { onAdd: (items: InventoryGroup[]) => void }) {
+  const params = useParams<{ id: string; inventory_type: string; host_id: string }>();
+
+  const toolbarFilters = useHostsGroupsFilters();
+  const tableColumns = useHostsGroupsColumns({ disableLinks: true });
```

Comment:
The sorting arrow in the name column header seems to be disabled

Reply:
This seems to be a framework level issue with all components using `MultiSelectDialog`. The bug tracking this work can be found here https://issues.redhat.com/browse/AAP-20202